Documentation/git-submodule.txt

```diff
@@ -358,6 +358,16 @@ the submodule itself.
 	If the key `submodule.$name.update` is set to `rebase`, this option is
 	implicit.
 
+--reset-hard::
+	This option is only valid for the update command.
+	Hard reset current state to the commit recorded in the	superproject.
+    If this option is given, the submodule's HEAD will not get detached
```

Comment:
Are you mixing tabs and spaces for indentation here?

If you want to make this a new paragraph in the man page, you can separate the new paragraph with a '+' and then not indenting it at all. (See the --reference NOTE below as that uses the same style)

Reply:
emacs played a trick on me. thanks for spotting, fixed now